common/dynamicconfig/constants.go

```diff
@@ -1178,6 +1178,13 @@ const (
 	// Allowed filters: ShardID
 	ReplicationTaskProcessorErrorRetryMaxAttempts
 
+	// MaxBufferedReplicationTasks is the max number of workflows for which we store in memory information
```

Comment:
Question: 
- Is this the hydration cache size? guess I'm not super sure which cache this is and it's relation to Vytautas' replication cache project.
- There was a (possibly separate/different) task to try and make the caches bound by Byte size, not workflow size (since some workflow executions are quite large). Is that something considered here?